src/TCommandLine.cpp

```diff
@@ -880,6 +885,12 @@ void TCommandLine::fillSpellCheckList(QMouseEvent* event, QMenu* popup)
 
 void TCommandLine::mousePressEvent(QMouseEvent* event)
 {
+    // Prevent selection, drag/drop of text in the command line when echo suppression is on
```

Comment:
Won't this also prevent bringing up a context menu to allow the user to select "paste" to insert a copied password *into* the command line? If so do we really want to do THAT?

Reply:
Done